nilearn/connectome/connectivity_matrices.py

```diff
@@ -120,10 +120,10 @@ def _geometric_mean(matrices, init=None, max_iter=10, tol=1e-7):
         Raise an error if the matrix is not symmetric positive definite of the
         same shape as the elements of matrices.
 
-    max_iter : int, optional
+    max_iter : int, optional, default is 10.
```

Comment:
I think the numpydoc convention is to put the default in the parameter description, rather than the datatype.